src/clj/rems/css/styles.clj

```diff
@@ -144,6 +144,26 @@
                    :border-color (get-theme-attribute :phase-bgcolor-completed "#ccc")
                    :color (get-theme-attribute :phase-color-completed :phase-color)}]]])
 
+(defn- generate-actions-float-menu
+  "The #actions floating menu can be too long for some screens. There is no clean solution for this in pure CSS
+  and to avoid yet another random JS-library we make the element scrollable on certain sizes.
+
+  We also do not want these styles to affect the mobile layout (i.e. more narrow than 992) where the actions
+  is at the bottom of everything and has any height available."
```

Comment:
this docstring probably could be reworded a bit now that the hack is gone

Reply:
to clarify, I think this no longer applies:

> There is no clean solution for this in pure CSS and to avoid yet another random JS-library we make the element scrollable on certain sizes

Reply:
:+1: I reworded it a bit to clarify